src/com/reason/esy/EsyCompiler.java

```diff
@@ -0,0 +1,96 @@
+package com.reason.esy;
+
+import com.intellij.execution.process.ProcessHandler;
+import com.intellij.execution.ui.ConsoleView;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.ui.SimpleToolWindowPanel;
+import com.intellij.openapi.vfs.VirtualFile;
+import com.intellij.openapi.wm.ToolWindow;
+import com.intellij.ui.content.Content;
+import com.reason.Compiler;
+import com.reason.CompilerType;
+import com.reason.Log;
+import com.reason.ProcessFinishedListener;
+import com.reason.ide.ORProjectManager;
+import com.reason.ide.console.CliType;
+import com.reason.ide.console.ORToolWindowProvider;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import javax.swing.*;
+import java.util.Optional;
+import java.util.Set;
+
+public class EsyCompiler implements Compiler {
+
+    private static final Log LOG = Log.create("esy.compiler");
```

Comment:
maybe the other way: ``compiler.esy`` ? 
We could enable debug for all compilers with ``ReasonML.compiler``  or one in particluar with ``ReasonML.compiler.esy``

Reply:
yup. that's a mistake. should be flipped around.